Formula/ffmpeg.rb

```diff
@@ -42,11 +32,11 @@ class Ffmpeg < Formula
   option "with-webp", "Enable using libwebp to encode WEBP images"
   option "with-zeromq", "Enable using libzeromq to receive commands sent through a libzeromq client"
   option "with-snappy", "Enable Snappy library"
-  option "with-dcadec", "Enable dcadec library"
   option "with-rubberband", "Enable rubberband library"
   option "with-zimg", "Enable z.lib zimg library"
   option "with-openh264", "Enable OpenH264 library"
   option "with-xz", "Enable decoding of LZMA-compressed TIFF files"
+  option "with-libebur128", "Enable libebur128 for EBU R128 loudness measurement"
```

Comment:
I derived the option description from FFmpeg's `configure`; now that I look at existing option descriptions though, "Enable *using* libebur128 for EBU R128 loudness measurement" could be better. But this is just a triviality.